api/src/resources/label_def.rs

```diff
@@ -46,3 +51,11 @@ pub struct NewLabelDefPretrained {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub name: Option<Name>,
 }
+
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+pub struct MoonFormFieldDef {
+    pub name: String,
+    pub kind: String,
+    pub required: bool,
+    pub r#type: String,
+}
```

Comment:
A comment about the API design -- although not the right place -- but `kind` and `type` are synonyms, I wished we called them something more explicit, e.g. `entity_kind`, `field_type`

Secondly, not sure how I feel about using `r#type` vs `#[serde(rename = "type")` -- fwiw, if we call `type` -> `field_type` (just in the struct for now), we won't be needing `r#`anyway